tgui/packages/tgui/interfaces/_FaxMachine.js

```diff
@@ -0,0 +1,286 @@
+import { useBackend, useSharedState, useLocalState } from '../backend';
+import { BlockQuote, Box, Button, Divider, Dropdown, Section, Stack, Tabs } from '../components';
+import { Window } from '../layouts';
+
+export const _FaxMachine = (props, context) => {
```

Comment:
Could you change this from `_FaxMachine` to just `FaxMachine`? The underscore is putting it at the top of the TGUI interfaces folder rather than sorting it alphabetically.
```suggestion
export const FaxMachine = (props, context) => {
```
<hr>

**Important:**
This also applies to the file name of `_FaxMachine.js`, and this part in `fax_machine.dm`:
```diff
/obj/machinery/fax_machine/ui_interact(mob/user, datum/tgui/ui)
	. = ..()
	ui = SStgui.try_update_ui(user, src, ui)
	if(!ui)
-		ui = new(user, src, "_FaxMachine", name)
+		ui = new(user, src, "FaxMachine", name)
		ui.open()
```